.github/workflows/v2.yaml

```diff
@@ -11,36 +11,43 @@ env:
   ECR_REPO: public.ecr.aws/u0u1j5s3/composer
 
 jobs:
-  build22:
-    name: Build v2.2
+  build-previous:
+    name: Build previous versions
     runs-on: ubuntu-latest
     timeout-minutes: 20
-    defaults:
-      run:
-        working-directory: '2.2'
```

Comment:
Can't use expressions here, that's why I pushed these configurations down.

See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrun.